jquery/jquery.d.ts

```diff
@@ -998,7 +998,22 @@ interface JQueryStatic {
      * @param objectN Additional objects containing properties to merge in.
      */
     extend(deep: boolean, target: any, object1?: any, ...objectN: any[]): any;
-
+    /**
+     * Merge the contents of two or more objects together into the first object.
+     *
+     * @param target An object that will receive the new properties if additional objects are passed in or that will extend the jQuery namespace if it is the sole argument.
+     * @param objects Rest parameter for use with spread operator for array of objects containing properties to merge in. 
+     */
+    extend(target: any, ...objects: any[]): any;
+    /**
+     * Merge the contents of two or more objects together into the first object.
+     *
+     * @param deep If true, the merge becomes recursive (aka. deep copy).
+     * @param target The object to extend. It will receive the new properties.
+     * @param objects Rest parameter for use with spread operator for array of objects containing properties to merge in.
+     */
+    extend(deep: boolean, target: any, ...objects: any[]): any;
```

Comment:
You need at least one source. That is why `extend(deep: boolean, target: any, object1?: any, ...objectN: any[]): any;` already exists